### PR TITLE
transmission_method could probably be enumerated through #define. Is neater.

### PR DESCRIPTION
**Pull request by Amunak**
Does what the freakin' comment says.

### CODE REVIEW

File: html/changelogs/transmission_method-could-probably-be-enumerated-through-define.would-be-neater.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: Amunak
+delete-after: True
+changes:
+  - refactor: "Refactors transmission_method values through #define. Is neater."
```
Comment (Geevies) [inline html/changelogs/transmission_method-could-probably-be-enumerated-through-define.would-be-neater.yml:4]: don't think players would care about this, i think this PR can go without a changelog

**Review by Geevies — APPROVED** [review]

File: html/changelogs/transmission_method-could-probably-be-enumerated-through-define.would-be-neater.yml (hunk 1 above)
Comment (Amunak) [reply html/changelogs/transmission_method-could-probably-be-enumerated-through-define.would-be-neater.yml:4]: Oh can you make a PR without a changelog? Wasn't entirely clear to me.

Comment (mikomyazaki) [reply html/changelogs/transmission_method-could-probably-be-enumerated-through-define.would-be-neater.yml:4]: Yeah, if it is something small like this that will have zero impact on players.

**Review by mikomyazaki — APPROVED** [review]

File: html/changelogs/transmission_method-could-probably-be-enumerated-through-define.would-be-neater.yml (hunk 1 above)
Comment (Amunak) [reply html/changelogs/transmission_method-could-probably-be-enumerated-through-define.would-be-neater.yml]: Alright, removed.

**Review by Arrow768 — APPROVED** [review]

**Comment by Amunak** [comment]
Conflict resolved, cart.dm was deleted by the PDA removal (I believe).

**Pull request merged**